python/activator/activator.py

```diff
@@ -955,17 +955,15 @@ def process_visit(expected_visit: FannedOutVisit):
                     _log.info("Running pipeline...")
                     try:
                         mwi.run_pipeline(expid_set)
-                        try:
-                            mwi.export_outputs(expid_set)
-                        except Exception as e:
-                            raise NonRetriableError("APDB and possibly alerts or central repo modified") \
-                                from e
                     except RetriableError:
                         # Do not export, to leave room for the next attempt
                         raise
                     except Exception:
                         _try_export(mwi, expid_set, _log)
                         raise
+                    else:
+                        _try_export(mwi, expid_set, _log)
+                        raise
```

Comment:
Did this `raise` actually work in testing? I would have expected some kind of error from there not being any exception...